host/cluster-down.yaml

```diff
@@ -257,6 +257,18 @@ spec:
         collectorName: "crictl-logs-haproxy-previous"
         command: "sh"
         args: ["-c", "crictl logs -p $(crictl ps -a --name haproxy -l --quiet) 2>&1"]
+    # Gathering hostname info to help troubleshoot scenarios where the hostname mismatch
```

Comment:
Would it be cleaner and perhaps easier to read the collected information if it were in the same file?

This spec should achieve that

```yaml
    - run:
        collectorName: "hostnames"
        command: "sh"
        args:
          - -c
          - |
            echo "\$HOSTNAME = $HOSTNAME"
            echo "hostname = $(hostname)"
            echo "/proc/sys/kernel/hostname = $(cat /proc/sys/kernel/hostname)"
            echo "uname -n = $(uname -n)"
```

Expected output

```
$HOSTNAME = evans-mac.local
hostname = evans-mac.local
/proc/sys/kernel/hostname = 
uname -n = evans-mac.local
```

Reply:
The envvar does not work in any way possible so we cannot add this one.